docs/getting-started.md

```diff
@@ -0,0 +1,196 @@
+# Getting Started with Piri
+
+This guide helps you choose the right Piri deployment path based on your specific needs and existing infrastructure.
+
+## 🤔 First, Let's Understand Your Needs
+
+Answer these questions to find your optimal setup path:
+
+### Question 1: Do you have existing Filecoin infrastructure?
+
+**No, I'm new to Filecoin storage**
+→ Continue to Question 2
+
+**Yes, I run Curio (or Lotus-Miner)**
+→ You should integrate Piri with your existing infrastructure
+→ 📖 **Follow**: [Piri with Curio Integration Guide](./integrations/piri-with-curio.md)
+
+### Question 2: What's your primary goal?
+
+**I want to join the Storacha network as a storage provider**
+→ You need the full stack (UCAN + PDP servers)
+→ 📖 **Follow**: [Full Stack Setup Guide](./integrations/full-stack-setup.md)
+
+**I want to build/test storage applications**
+→ Continue to Question 3
+
+**I'm evaluating the technology**
+→ 📖 **Start with**: [Architecture Overview](./architecture.md)
+→ 📖 **Then try**: Local development setup in [Full Stack Guide](./integrations/full-stack-setup.md)
+
+### Question 3: Which component do you need?
```

Comment:
Who is Question 3 oriented towards? Devs contributing to Piri no? Or maybe people looking to build on top of it? Either way, I'd move it to a Contributing.md or somewhere else. It's confusing.